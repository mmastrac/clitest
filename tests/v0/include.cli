#!/usr/bin/env clitest --v0

include "include/included.cli";

$ echo $FROM_INCLUDED
! from included

# This should escape the include block
$ echo "reject"
%EXPECT_FAILURE
*
