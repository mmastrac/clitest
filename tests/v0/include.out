Running <root>/include.cli ...

set FROM_INCLUDED from included

echo $FROM_INCLUDED
---
from included
---
✅ OK

echo "reject"
---
reject
---
✅ OK

echo "hello"
---
hello
---
✅ OK

<root>/include.cli PASSED
